contracts/zero-ex/contracts/src/features/ERC721OrdersFeature.sol

```diff
@@ -278,41 +273,37 @@ contract ERC721OrdersFeature is
         _validateBuyOrder(
             buyOrder,
             buyOrderSignature,
-            msg.sender, // TODO: maybe allow sellOrder.taker == buyOrder.maker, vice versa
+            sellOrder.maker,
```

Comment:
If we're going to do this we should also pass in `buyOrder.maker` to `_validateSellOrder`